src/modules/emailMethods.ts

```diff
@@ -134,9 +134,7 @@ export const assignSectionEmail = async (data: AssignSectionEmailData) => {
         start: exception.date,
         description: data.description || exception.eventDetails.description,
         duration: exception.eventDetails.duration,
-        summary: data.sectionTypeTitle
-            ? `${data.sectionTypeTitle} ${tr('with')} ${data.candidateName}`
-            : `${tr('Interview with')} ${data.candidateName}`,
+        summary: `${data.sectionTypeTitle} ${tr('with')} ${data.candidateName}`,
```

Comment:
Why did you change it?

Reply:
sectionTypeTitle is obligatory field, so this ternar doesn't make sense